tests/generic_config_updater/field_operation_validator_test.py

```diff
@@ -14,6 +14,85 @@
 
 class TestValidateFieldOperation(unittest.TestCase):
 
+    @patch("generic_config_updater.field_operation_validators.read_statedb_entry", mock.Mock(return_value=""))
+    def test_port_config_update_validator_valid_speed_no_state_db(self):
+        patch_element = {"path": "/PORT/Ethernet3", "op": "add", "value": {"speed": "234"}}
+        assert generic_config_updater.field_operation_validators.port_config_update_validator(patch_element) == True
+    
+    @patch("generic_config_updater.field_operation_validators.read_statedb_entry", mock.Mock(return_value=""))
+    def test_port_config_update_validator_invalid_speed_no_state_db(self):
+        patch_element = {"path": "/PORT/Ethernet3", "op": "add", "value": {"speed": "xyz"}}
+        assert generic_config_updater.field_operation_validators.port_config_update_validator(patch_element) == False
```

Comment:
Looks like it hit true in unit test.

Reply:
Yes thanks, I changed this unit test to return existing speeds to validate against in StateDB. So now, False is returned because in this case, we attempt to typecast the value to an int. 

Previously, the unit test did not return any values in StateDB to validate against, so we never reached the typecast error being thrown. In this case, the invalid speed is caught by YANG validation so it was correct to return True before I changed this unit test.